src/main/features/response/views/reject-all-of-claim.njk

```diff
@@ -0,0 +1,51 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, radioGroup, saveAndContinueButton, errorSummary %}
+{% from "table.njk" import tableStart, tableEnd, row, rowWithMultipleValue %}
+
+{% set title = t("Money claim") %}
+
+{% set heading = t("I reject all of the claim") %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+
+      <form method="post" novalidate>
+        {{ csrfProtection(csrf) }}
+
+        <div class="form-group {% if error %} form-group-error {% endif %}">
+
+            <legend class="visually-hidden">{{ t("I reject all of the claim") }}</legend>
+            {% if error %}<span class="error-message">{{ t(error) }}</span>{% endif %}
+        {{
+        radioGroup(
+        label = "Why do you reject all the claim?",
+        name = "option",
+        options = [
+        {
+          label: "I've already paid the claim in full",
+          value: "alreadyPaid",
+          hint: ""
+        },
+        {
+          label: "I dispute the claim",
+          value: "dispute",
+          hint: ""
+        },
+        {
+          label: 'Defend now and also make a counterclaim',
+          value: "counterClaim",
+          hint: 'You’ll need to complete the defence and counterclaim form.'
+        }
+        ],
+        form = form
+        )
+        }}
+          <div class="form-group">
```

Comment:
We do not put buttons into `form-group` div. Any reason you did it? If not please remove.